.github/DISCUSSION_TEMPLATE/new.yml

```diff
@@ -0,0 +1,69 @@
+name: New Discussion
+description: Please fill out the following information to create a new discussion.
+  Make sure to provide clear and concise details to help others understand and address your concerns effectively.
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Problem description
+      description: A clear and concise description of what the problem is
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: How to reproduce
+      description: Steps to reproduce the behavior
+      value: |
+        <!--
+        1. Go to '...'
+        2. Change '....'
+        3. Run '....'
+        4. See error
+        -->
+    validations:
+      required: true
```

Comment:
Note about all of these:
In the bug report template, it makes sense to have them required, but here I think they should be optional given that the subject of a discussion could be "anything"?

Reply:
I agree, this does not sound like the right template for a generic "New Discussion" form.